lib/livebook/hubs/personal.ex

```diff
@@ -67,7 +68,7 @@ defmodule Livebook.Hubs.Personal do
     personal
     |> cast(attrs, @fields)
     |> validate_required(@fields)
-    |> validate_change(:secret_key, fn :secret_key, secret_key ->
+    |> validate_change(:secret_key, fn :secret_key, @prefix <> secret_key ->
```

Comment:
They may paste something without the prefix, so we should validate the prefix too.